src/testrender/optixraytracer.cpp

```diff
@@ -486,6 +499,11 @@ OptixRaytracer::make_optix_materials()
             OIIO::Filesystem::write_text_file(filename, osl_ptx);
         }
 
+        void* interactive_params = nullptr;
+        shadingsys->getattribute(groupref.get(), "device_interactive_params",
+                                 TypeDesc::PTR, &interactive_params);
+        material_interactive_params.push_back(interactive_params);
```

Comment:
My only comment is this double bookkeeping renderer/shadingsys for the pointers, but I can't think of a better approach that we can get this week. The only thing that the renderer does with these pointers is passing them back, right?
In the future the entry calls could receive an opaque handle where you can track things like this that OSL fully owns.